index.js

```diff
@@ -920,6 +920,20 @@ const fileType = input => {
 		};
 	}
 
+	if (check([0x4C, 0x00, 0x00, 0x00, 0x01, 0x14, 0x02, 0x00, 0x00, 0x00, 0x00, 0x00, 0xC0, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x46])) {
+		return {
+			ext: 'lnk',
+			mime: 'application/x.ms.shortcut'
```

Comment:
This is not the correct MIME type.

Reply:
There is no mention of specific mime type for them anywhere I tried.
x mime types are not registered and don't signal anything by their format.
I used scheme people usually apply to Microsoft and Apple related formats.

`file` gives `application/octet-stream`for them, but for the purposes of say Content-Type it isn't correct, it should be missing instead.

So, there are 3 options:
- using x tree and coming up with new type, at least for the people who just check it programmatically. It will also be correct for supplying to other applications, as it isn't registered nor known type.
- using `undefined`. It is correct for the purposes of Content-Type, but doesn't help with programmatic use, doesn't happen anywhere in the library and changes type definitions for typescript.
- using `application/octet-stream`, like `file`, which is incorrect and might trigger unwanted behavior in browsers for example.
- using `application/unknown` which is correct, but doesn't signal anything to api users, just as `undefined`

Reply:
The one mention in the link is using dashes. You're using `.`

Reply:
Using `x-` is against modern standards.
'.' is recommended delimiter for parts of subtype.
Unregistered tree is specifically defined by `x.`, not `x-` nor just `x`.

Reply:
Ah ok. I didn't know that.